File: Python/Anshika Srivastava/day9q2.py
import string

sentence = input("Enter sentence\n")
sentence = sentence.lower()
alphabet = string.ascii_lowercase

pangram = True

for char in alphabet:
    if char not in sentence:
        pangram = False
        break

if pangram:
    print("The sentence is a pangram.")
else:
    print("The sentence is not a pangram.")

File: Python/Supragya/Day9_Q2.c
# Q2 Write a Python program that checks if a given sentence is a pangram (contains every letter of the alphabet).
#  Display an appropriate message.
import string
def is_pangram(sentence):
    sentence = sentence.lower()
    letters = set()
    for char in sentence:
        if char.isalpha():
            letters.add(char)
    return len(letters) == 26

sentence = input("Enter a sentence: ")
if is_pangram(sentence):
    print("The sentence is a pangram.")
else:
    print("The sentence is not a pangram.")
